tests/unit/test_eq.py

```diff
@@ -377,4 +377,4 @@ def test_build_display_address_prems(self):
         ]:
             self.assertEqual(
                 eq.EqPayloadConstructor.build_display_address(attributes),
-                expected)
+                expected)
```

Comment:
The PEP8 style guide requires every line should end with a newline, because the newline is considered a line terminator. So this doesn't conform to PEP8.

Reply:
Fixed